dnsmasq-plan/plan.sh

```diff
@@ -0,0 +1,31 @@
+pkg_name=dnsmasq
+pkg_origin=core
+pkg_version=2.80
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_description="Provides network infrastructure for small networks"
+pkg_upstream_url="http://www.thekelleys.org.uk/dnsmasq/doc.html"
+pkg_license=("GPL")
+pkg_source="http://www.thekelleys.org.uk/dnsmasq/dnsmasq-${pkg_version}.tar.gz"
+pkg_shasum="9e4a58f816ce0033ce383c549b7d4058ad9b823968d352d2b76614f83ea39adc"
+pkg_deps=(
+  core/glibc
+)
+pkg_build_deps=(
+  core/gcc
+  core/make
+)
+pkg_bin_dirs=(
+  bin
+)
+pkg_svc_run="dnsmasq -x $svc_path/dnsmasq.pid -C $pkg_svc_config_path/dnsmasq.conf"
```

Comment:
I _think_ we need to pass `--no-daemon` to dnsmasq,  otherwise the run hook goes into a loop trying restart the process.

Reply:
I added it and tested the plan in _standalone_ mode and it worked. I haven't yet tested it as a dependency in another plan though.